js_modules/dagster-ui/packages/ui-core/src/asset-graph/SidebarAssetInfo.tsx

```diff
@@ -61,14 +61,15 @@ export const SidebarAssetInfo = ({graphNode}: {graphNode: GraphNode}) => {
   const {lastMaterialization} = liveData || {};
   const asset = data?.assetNodeOrError.__typename === 'AssetNode' ? data.assetNodeOrError : null;
 
-  const recentEvents = useRecentAssetEvents(
-    asset?.assetKey,
-    1,
-    MaterializationHistoryEventTypeSelector.MATERIALIZATION,
+  const recentEvents = useRecentAssetEvents(asset?.assetKey, 1, [
+    AssetEventHistoryEventTypeSelector.MATERIALIZATION,
+    AssetEventHistoryEventTypeSelector.OBSERVATION,
+  ]);
+  const materializations = recentEvents.events.filter(
```

Comment:
not sure if filtering based on `__typename` is the right thing to do here

Reply:
yeah thats fine